.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
Actually, maybe not. ES6 arrow functions were there, but I didn't check it after a rebase on top of the latest `master`.